### PR TITLE
Whiteboard page view

### PR DESCRIPTION
**Pull request by jkeat**
Description
-----------
- tall bounded document (1000 x 2000)
- zooms to fill width on all devices
- vertically scrollable on desktop
- remove pan tool
- on mobile, still allow multi-touch panning & zooming (this is the only time it’s possible to see outside of the drawable area)
- grid background

Developer self-review checklist
-------------------------------
- [ ] Potentially confusing code has been explained with comments
- [ ] No warnings or errors have been introduced; all known error cases have been handled
- [ ] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [ ] All edge cases have been addressed

### CODE REVIEW

File: src/views/SessionView/Whiteboard.vue [hunk 1]
```diff
@@ -279,6 +280,7 @@ export default {
       autoPickTool: false,
       autoPickToolText: false,
       defaultBrushWidth: 5,
+      allowZoom: false,
```
Comment (jkeat) [inline src/views/SessionView/Whiteboard.vue:283]: disable the pan tool's ability to zoom; all zooming is done ourselves via zwibbler api

**Review by treykris — DISMISSED** [review]
Awesome, this works great!

**Comment by jkeat** [comment]
Update: this breaks multitouch zoom/pan on mobile, i think due to the `allowZoom: false` change

**Comment by jkeat** [comment]
todo:
- enable page view, make it relatively small
- don't allow viewing outside of page bounds (min zoom = no bigger than page width/height)

File: src/views/SessionView/Whiteboard.vue [hunk 2]
```diff
@@ -310,6 +309,9 @@ export default {
       // Set brush tool to default tool
       this.useBrushTool();
 
+      // Zoom to full width
+      this.zwibblerCtx.setViewRectangle({ x: 0, y: 0, width: 1000, height: 1 });
```
Comment (treykris) [inline src/views/SessionView/Whiteboard.vue:313]: The view needs to be set again once connected to the Zwibbler collab server?

Comment (jkeat) [reply src/views/SessionView/Whiteboard.vue:313]: Yeah, if you remove this then on re-connection (returning to session from dashboard) it'll zoom to the wrong view

**Review by treykris — APPROVED** [review]
Nice work!!

**Comment by jkeat** [comment]
Closing in favor of https://github.com/UPchieve/web/pull/517

**Pull request closed without merging**